src/coreclr/nativeaot/Runtime/CMakeLists.txt

```diff
@@ -234,7 +234,7 @@ add_definitions(-DFEATURE_BASICFREEZE)
 add_definitions(-DFEATURE_CONSERVATIVE_GC)
 add_definitions(-DFEATURE_CUSTOM_IMPORTS)
 add_definitions(-DFEATURE_DYNAMIC_CODE)
-add_compile_options($<$<OR:$<CONFIG:Debug>,$<CONFIG:Checked>>:-DFEATURE_GC_STRESS>)
+add_compile_definitions($<$<OR:$<CONFIG:Debug>,$<CONFIG:Checked>>:FEATURE_GC_STRESS>)
```

Comment:
This fixes "unresolved external symbol `RhpGcStressHijack`" error when using the Windows ARM64 debug package. When we preprocess assembler files for ARM and ARM64, compile definitions are passed to the preprocessor, but compile options are not.